pkg/kademlia/kademlia_test.go

```diff
@@ -218,116 +218,6 @@ func TestRefresh(t *testing.T) {
 	assert.True(t, ts1.Equal(ts2))
 }
 
-func TestGetNodes(t *testing.T) {
```

Comment:
oh shoot, sorry, we should keep this, we just need to rename to FindNear

Reply:
Updated 👍